src/java.base/share/classes/java/lang/Enum.java

```diff
@@ -25,6 +25,14 @@
 
 package java.lang;
 
+import java.lang.SuppressWarnings;
```

Comment:
@mernst 
Here again no need to have this `import`.
Though I can't find any extra annotation here like in the case of `@FunctionalInterface`, which is good.